ASM/c/en_wonderitem.c

```diff
@@ -0,0 +1,151 @@
+#include <stdint.h>
+#include "z64.h"
+#include "en_wonderitem.h"
+#include "get_items.h"
+
+static colorRGBA8_t sEffectPrimColorRed = { 255, 0, 0, 0 };
+static colorRGBA8_t sEffectPrimColorGreen = { 0, 255, 0, 0 };
+static colorRGBA8_t sEffectPrimColorBlue = { 0, 0, 255, 0 };
+static colorRGBA8_t sEffectPrimColorYellow = { 255, 255, 0, 0 };
+static colorRGBA8_t sEffectPrimColoMagenta = { 255, 0, 255, 0 };
+static colorRGBA8_t sEffectPrimColorCyan = {0, 255, 255, 0 };
+static colorRGBA8_t sEffectEnvColor = { 255, 255, 255, 0 };
+//static z64_xyzf_t sEffectVelocity = { 0.0f, 0.1f, 0.0f };
+//static z64_xyzf_t sEffectAccel = { 0.0f, 0.01f, 0.0f };
+static z64_xyzf_t sEffectVelocity = { 0.0f, 0.5f, 0.0f };
+static z64_xyzf_t sEffectAccel = { 0.0f, 0.5f, 0.0f };
+
+
+extern uint16_t drop_collectible_override_flag;
+extern uint16_t CURR_ACTOR_SPAWN_INDEX;
+
+void EnWonderitem_AfterInitHack(z64_actor_t* this, z64_game_t* globalCtx)
+{
+    if(this->main_proc == NULL)
+        return;
+    if(this->actor_id != 0x112)
+        return;
+
+    EnWonderItem* wonderitem = (EnWonderItem*)this;
+    wonderitem->overridden = 0;
+
+    EnItem00 dummy;
+    dummy.actor.actor_id = 0x15;
+    dummy.actor.rot_init.y = this->rot_init.y; //flag was just stored in z rotation
```

Comment:
The comment should say y rotation, right?

Reply:
Yes. This will change with the new potsanity PR just depends which gets merged first.